NBXplorer/Startup.cs

```diff
@@ -54,7 +55,7 @@ public void ConfigureServices(IServiceCollection services)
 			builder.AddNewtonsoftJson(options =>
 			{
 				options.SerializerSettings.ContractResolver = new CamelCasePropertyNamesContractResolver();
-				new Serializer(null).ConfigureSerializer(options.SerializerSettings);
+				new Serializer(new NBXplorerNetwork(NBitcoin.Bitcoin.Instance, NetworkType.Mainnet)).ConfigureSerializer(options.SerializerSettings);
```

Comment:
no, you can't do this.

Reply:
I added this after bumping NBitcoin. Apparently the serializers from NBitcoin now require that the network is not null?

Reply:
Checking why, but this is a bug of NBXplorer not of NBitcoin. The NBitcoin 5.0 became less forgiving into serializing stuff without knowing the Network. And for good reason, as you, as it catched the fact that in liquid, you can't possibly serialize properly the TxOut properly.

Reply:
I pushed a fix on master, rebase your PR.

Reply:
done

Reply:
so can you remove this change?

Reply:
ah I thought I did, will do now

Reply:
done